### PR TITLE
[DEVOPS-2718] service provider list update

### PR DESCRIPTION
**Pull request by ppodbielski-cloudentity**
## Implementation details (internal)
- edit service provider list to fission, docker, hybrid
- removing flag: `fission.enabled` and replacing it with `faas.provider`

## Information for QA

- [x] Is QA testing required?
- [ ] Does PR contain unit tests?
- [ ] Should QA create E2E tests for the change?

### CODE REVIEW

**Review by lgonczarik — APPROVED** [review]

File: charts/acp/values.yaml [hunk 1]
```diff
@@ -592,17 +592,14 @@ workers:
   ##
   # podDisruptionBudget: {}
 
-## Fission chart configuration
+## Available FaaS providers: fission, docker, hybrid
```
Comment (artursmolarek) [inline charts/acp/values.yaml:595]: Please add more context here and explain the differences between these modes.

**Review by artursmolarek — CHANGES_REQUESTED** [review]

**Review by mbilski — CHANGES_REQUESTED** [review]
let's make it non breaking change

**Review by jkotiuk — APPROVED** [review]

File: charts/acp/values.yaml (hunk 1 above)
Comment (plandzwojczak-cloudentity) [reply charts/acp/values.yaml:597]: You're pointing it to the doc which uses fission: enabled https://cloudentity.com/developers/deployment-and-operations/configure/configure-fission-for-faas/#fission-integration-for-faas
In this PR you're removing this option. I believe this doc should be updated according to this changes

**Comment by ppodbielski-cloudentity** [comment]
> is this backward compatible now? Will it work if I just update the helm without changing the configuration? Is there a need for faas.enabled flag? Woudn't empty faas provider mean the same?

QA team is checking backward compatible
`faas.enabled` unfortunately, this flag is needed to keep backward comp.

**Review by mbilski — APPROVED** [review]

**Review by artursmolarek — APPROVED** [review]

**Pull request merged**